ganga/GangaCore/Core/MonitoringComponent/MonitoringService.py

```diff
@@ -114,7 +114,10 @@ def _check_backend(self, backend):
 
     def _schedule_next_backend_check(self, backend):
         backend_name = getName(backend)
-        pRate = config.get(backend_name, config['default_backend_poll_rate'])
+        if backend in config:
+            pRate = config[backend_name]
+        else:
+            pRate = config['default_backend_poll_rate']
```

Comment:
All backends that we know about should have a corresponding config. So I do not htink this check is required?

Reply:
@egede There are a couple of testing backends like the one I created and TestSavannah which I didn't find in the `PollThread` config. I would say it is good to have a fallback even it is redundant 99% of the time but if you would prefer it I can remove that check.

That being said, there was another issue with this code. Using the `default_backend_poll_rate` as a fallback (which is 30 seconds), caused [this integration test](https://github.com/ganga-devs/ganga/blob/229dc06a38e6093a85f0bcda54b9126d80e76a50/ganga/GangaCore/test/GPI/TestTimeStamps.py#L157) to fail and I am not sure why ([here are the workflow logs](https://github.com/ganga-devs/ganga/actions/runs/3136206338/jobs/5092866735)). The test now passes by using the `base_poll_rate` which is 2 seconds as a fallback. So do the GPI tests use a different backend to submit to? And if so could it be that the 30 second default poll rate is too high?

Reply:
Yes, decreasing the base poll rate to 15 seconds did fix that test. Should I leave it in as the default fallback poll rate?